magento-research/pwa-studio
# Category RootComponent Simple Re-Factor

## Description

Re-factor Category RootComponent and all components inside component directory to be functional components that use hooks.

## Related Issue





Closes #975

## Verification Steps

1. Go to the Bottoms Category
2. Verify that a product list renders

## How Have YOU Tested this?



Manually ran steps above and ran test suite

## Screenshots / Screen Captures (if appropriate):

## Proposed Labels for Change Type/Package




## Checklist:
